angular.json

```diff
@@ -195,7 +195,8 @@
     }
   },
   "cli": {
-    "defaultCollection": "@nrwl/angular"
+    "defaultCollection": "@nrwl/angular",
+    "analytics": false
```

Comment:
I keep getting a "do you want to submit analytics to Angular..."-text when running `ng start`, resulting in this change appearing "over and over again".

I made a decision.... Nope, we won't submit analytics, and we'll stop getting "git changes".